Cargo.toml

```diff
@@ -15,18 +15,18 @@ name = "tessellation"
 path = "src/lib.rs"
 
 [dependencies]
-alga = "0.9"
-nalgebra = "0.22"
-rand = "0.7"
-rayon = "1.2"
-once_cell = "1.4"
-bbox = "0.11"
-num-traits = "0.2"
+bbox = "0.11.2"
```

Comment:
Wouldn't just major+minor be fine?

Reply:
I'm using the default conventions of the `cargo-edit` tool which was what I used to automate this change.  I suppose it might help ensure that users of this crate get at least the version of a dependency that was used to run the test suite.  Imagine if `0.5.2` is buggy, `0.5.3` is fine, and the tests of this crate were tested against `0.5.3`.  Setting the patch version ensures that users don't accidentally get the `0.5.2` version.

Reply:
see https://github.com/hmeyer/bbox/pull/3#discussion_r649721547